wm/screens.go

```diff
@@ -46,36 +46,38 @@ func (xsp *x11ScreensProvider) Scale() float32 {
 }
 
 func (xsp *x11ScreensProvider) ScreenForWindow(win desktop.Window) *desktop.Screen {
-	if len(xsp.screens) > 1 {
```

Comment:
Haha, when you made this change I realised - this function should just call ScreenForGeometry with the window geometry surely?